src/constants.ts

```diff
@@ -7,7 +7,7 @@ export const LIMIT_ARCHIVE_SIZE = 100 * 1024 * 1024; // 100MB
 export const AUTO_UPDATE_INTERVAL_MS = 60 * 60 * 1000;
 export const WINDOWS_TITLEBAR_HEIGHT = 32;
 export const ABOUT_WINDOW_WIDTH = 284;
-export const ABOUT_WINDOW_HEIGHT = 284;
+export const ABOUT_WINDOW_HEIGHT = 350;
```

Comment:
We could possibly drop it to 340 without triggering the scrollbar if the window feels a bit big